### PR TITLE
[IMP] finance: avatax version 17 updates

### PR DESCRIPTION
**Pull request by tiku-odoo**
Task: https://www.odoo.com/web#id=3905581&cids=3&menu_id=4720&action=333&active_id=3835&model=project.task&view_type=form

### CODE REVIEW

**Comment by tiku-odoo** [comment]
@ksc-odoo 

This doc is ready for your review when you have a moment. 

Thanks! 

Tim

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:178]:
```suggestion
     - Includes tax calculation in Odoo *Inventory*.
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:195]:
```suggestion
     - This module includes the required features for tax calculation for services in the Brazil
       localization.
   * - :guilabel:`Avatax Brazil Sale for Services`
     - `l10n_br_edi_sale_services`
     - This module includes the required features for tax calculation for the sale of services in
       the Brazil localization.
   * - :guilabel:`Test SOs for the Brazilian AvaTax`
     - `l10n_br_test_avatax_sale`
     - This module includes the required features for test sales orders in the Brazil localization.
```
just an optional way to slightly enhance the "flow" of the explanations

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:197]:
```suggestion
Click the :guilabel:`Install` button on the module labeled, :guilabel:`Avatax`: `account_avatax`.
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:181]:
```suggestion
     - Includes tax calculation features between the *Amazon Connector* and Odoo.
```
this suggestion is based on how you format the Amazon Connector below

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:204]:
```suggestion
Should *AvaTax* be needed for geo-localization, or with the *Amazon Connector*, then install those
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:206]:
```suggestion
and :guilabel:`Amazon/Avatax Bridge`, respectively.
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:171]:
```suggestion
     - This module includes the features required for integration of *AvaTax* into geo-localization in
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:209]:
```suggestion
   To install *AvaTax* for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil` module.
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:210]:
```suggestion
   This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for Services`.
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:220]:
```suggestion
fields in the :guilabel:`Taxes` section is where the *AvaTax* configurations are made, and the
credentials are entered.
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:222]:
```suggestion
First, tick the checkbox to the left of the :guilabel:`AvaTax` settings, to activate *AvaTax* on the
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:223]:
```suggestion
database. This is a quick, convenient way to activate and deactivate *AvaTax* tax calculation on the
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:388]:
```suggestion
   Additionally, there is a :guilabel:`Tax Mapping` tab and :guilabel:`Account Mapping` tab in the
```

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:389]:
```suggestion
   :guilabel:`Automatic Tax Mapping (AvaTax)` fiscal position, where mapping for products can also
```

File: content/applications/finance/accounting/taxes/avatax/avatax_use.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax/avatax_use.rst:12]:
```suggestion
Automatically calculate taxes on Odoo quotations and invoices with *AvaTax* by confirming the
```

File: content/applications/finance/accounting/taxes/avatax/avatax_use.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax/avatax_use.rst:14]:
```suggestion
:guilabel:`Compute Taxes` button, while these documents are in draft mode.
```

Comment (ksc-odoo) [reply content/applications/finance/accounting/taxes/avatax/avatax_use.rst:14]: do you think 'draft mode' or 'draft stage' is more accurate? Totally your call...

File: content/applications/finance/accounting/taxes/avatax/avatax_use.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax/avatax_use.rst:17]:
```suggestion
   Clicking the :guilabel:`Compute Taxes` button recalculates taxes, if any product lines are edited
```

File: content/applications/finance/accounting/taxes/avatax/avatax_use.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax/avatax_use.rst:47]: Do you think it would be beneficial to add some instructions to these explanations (e.g. "Click the...")... or do you think that is pretty self-explanatory? Just something to consider during your revisions. 🤔

File: content/applications/finance/accounting/taxes/avatax/avatax_use.rst
Comment (ksc-odoo) [inline content/applications/finance/accounting/taxes/avatax/avatax_use.rst:98]:
```suggestion
invoice. Add the product discount, and set the :guilabel:`Price` to either a positive or negative
```

**Review by ksc-odoo — APPROVED** [review]
Hey @tiku-odoo -- just finished my review. Nicely done. When it comes to my suggestions of italicizing the *AvaTax* mentions, feel free to take or leave those as you see fit. However, it would be best to pick one route, and keep things uniform throughout the doc. May even want to double-check with Sam about whether or not that kinda thing should be italicized...which, personally, I *think* it should be because its the name of an app/software, which we, otherwise, always format in that way. But, again, that's totally up to you. Other than that, once you implement the necessary changes, feel free to tag this for the next stage. Thanks! 👍

**Comment by tiku-odoo** [comment]
@samueljlieber 

Sam, 

This PR is ready for your review, just a few minor edits to the screenshots and adding in Brazil for 17. 

Two docs edited:

avatax.rst
avatax/avatax_use.rst

Thanks, 
Tim :+1:

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (tiku-odoo) [inline content/applications/finance/accounting/taxes/avatax.rst:211]:
```suggestion
.. tip::
   To install the *AvaTax* localization for Brazil, click :guilabel:`Install` on the :guilabel:`AvaTax Brazil`
   module. This action installs :guilabel:`AvaTax Brazil` and :guilabel:`AvaTax Brazil for
   Services`. Install the additional Brazil *Avatax* modules as needed.
   
  .. seealso::
    Reference the :doc:`Brazilian localization documentation <../../fiscal_localizations/brazil>` for more specifics on Brazilian localization. 
```

**Review by tiku-odoo — COMMENTED** [review]
@samueljlieber 

I propose this change to the document and leave everything else as is. 

@StraubCreative 
Would you prefer Brazil to be included in the main Avatax documentation, which would include all the locales Avatax is available in and link it to the Brazil localization doc? Or would you like to leave Brazil off the Avatax document and have the documentation fragmented according to localization for this integration?

Currently, it's documented in the main AvaTax document, and Sam is working separately on the Brazilian PR. I personally think it would benefit to have it in both places so that the main AvaTax document is complete with all the locales Odoo offers AvaTax in. 

See the attached changes. 

Thanks, Tim

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (samueljlieber) [reply content/applications/finance/accounting/taxes/avatax.rst:211]: Thanks @tiku-odoo, and curious on your thoughts here @StraubCreative!

Here is the PR @vbe-odoo and I are working on, that details the l10n_br electronic invoicing & tax computation flow https://github.com/odoo/documentation/pull/8784/. 

Just my two cents– I think it is a good idea to list __all__ of the AvaTax modules that are available, even Brazil, in this main AvaTax document, but leave the local-specific install/config instructions to the fiscal localization docs, even though Brazil is the only local besides USA & Canada that Odoo supports for AvaTax.

Here is what I am suggesting, instead of the tip admonition that details Brazil AvaTax modules:
```suggestion
.. seealso::
   For localization specific *AvaTax* instructions, view the following :doc:`fiscal localization
   <../../fiscal_localizations>` documentation:

   - :doc:`../../fiscal_localizations/brazil`
```
Granted, this can be sorted out in another [IMP] PR on either this avatax.rst doc, or the brazil.rst fiscal localization doc so that we can get this important information live for 17.0+.

File: content/applications/finance/accounting/taxes/avatax.rst
Comment (samueljlieber) [inline content/applications/finance/accounting/taxes/avatax.rst:342]: No italics formatting in a guilabel, it doesnt render :)
```suggestion
(AvaTax)`. Click it to open *AvaTax's* fiscal position configuration page.
```

**Review by samueljlieber — APPROVED** [review]
Hi @tiku-odoo, thank you for hard work on this PR! Your improvements look good to me, I am approving with a comment and delegating merge to you. Let's try to reach a decision on the [local discussion](https://github.com/odoo/documentation/pull/9341/files#r1603570858) merge, we can chat in discord :)
..
@robodoo delegate=tiku-odoo

**Comment by tiku-odoo** [comment]
@robodoo r+

**Pull request closed without merging**
